src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -256,7 +278,8 @@ private static bool IsCodeActionNotSupportedByLSP(IUnifiedSuggestedAction sugges
             CodeActionOptionsProvider fallbackOptions,
             ICodeFixService codeFixService,
             ICodeRefactoringService codeRefactoringService,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken,
+            string? fixAllScope = null)
```

Comment:
make non optional.  it will be good for callers to be clear what they are passing for this.